### PR TITLE
Set up linter and pre commit

### PR DESCRIPTION
**Pull request by mskab**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- repo: https://github.com/pycqa/flake8
+  rev: 4.0.1
+  hooks:
+    - id: flake8
+      exclude: ^(.*\b(migrations)\b.*|.*\b(venv)\b.*)$
+      additional_dependencies:
+        - flake8-bugbear
+        - flake8-comprehensions
+        - flake8-simplify
```
Comment (mskab) [inline .pre-commit-config.yaml]: Main changes for that pr

File: Pipfile [hunk 2]
```diff
@@ -12,6 +12,9 @@ psycopg2 = "*"
 alembic = "*"
 pydantic = {extras = ["email"], version = "*"}
 passlib = {extras = ["bcrypt"], version = "*"}
+pre-commit = "*"
+flake8-fastapi = "*"
+black = "*"
 
 [dev-packages]
```
Comment (mskab) [inline Pipfile:22]: Main changes for that pr

**Review by mikaserova — COMMENTED** [review]
Add also isort and pylint

**Comment by mskab** [comment]
> Add also isort and pylint

I added it

**Review by mikaserova — APPROVED** [review]

**Pull request merged**